static/open-specs/liveobjects.yaml

```diff
@@ -0,0 +1,641 @@
+          schema:
+            type: boolean
+            default: false
+      responses:
+        '200':
+          description: List of retrieved objects or object references.
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - type: array
+                    items:
+                      type: object
+                      required:
+                        - objectId
+                      properties:
+                        objectId:
+                          type: string
+                        map:
+                          type: object
+                          properties:
+                            entries:
+                              type: object
+                              additionalProperties:
+                                type: object
+                                properties:
+                                  data:
+                                    type: object
+                                    oneOf:
+                                      - properties:
+                                          string:
+                                            type: string
+                                      - properties:
+                                          objectId:
+                                            type: string
+                        counter:
+                          type: object
+                          properties:
+                            data:
+                              type: object
+                              properties:
+                                number:
+                                  type: number
+                  - type: array
+                    items:
+                      type: string
+              examples:
+                objectArray:
+                  summary: Full response with object values
+                  value:
+                    - objectId: "map:YIffJYRAP2k2e7ZP+xzequ9c5kDu1LfI/sEOKoWHvv4@1742479683000"
+                      map:
+                        entries:
+                          myMapKey: { data: { string: "my map value" }}
+                          myBoolean: { data: { boolean: true }}
+                          myEncodedKey: { data: { bytes: "TGl2ZU9iamVjdHMgaXMgYXdlc29tZQo=", encoding: "base64" }}
+                    - objectId: "counter:Nz1ZiNjqsDfkDjA61xarinqpWsqEGAAw2mzWWtvX2b8@1742481614000"
+                      map:
+                        entries:
+                          counter: { data:{ number: 5 }}
+                          myObjectRef: { data: { objectId: "counter:Nz1ZiNjqsDfkDjA61xarinqpWsqEGAAw2mzWWtvX2b8@1742481614000" }}
+                refArray:
+                  summary: Just object references
+                  value:
+                    - "root"
+                    - "map:YIffJYRAP2k2e7ZP+xzequ9c5kDu1LfI/sEOKoWHvv4@1742479683000"
+                    - "counter:Nz1ZiNjqsDfkDjA61xarinqpWsqEGAAw2mzWWtvX2b8@1742481614000"
```

Comment:
It wasn't super clear to me from the rendered doc that the difference in these two examples was the inclusion or not of `values=true` query param. Maybe we can reference that somewhere?